packages/proposals

```diff
@@ -0,0 +1,53 @@
+1 PKG_PRIORITY
+
+None of the current scripts use the PKG_PRIORITY variable.
+All but three packages set PKG_PRIORITY="optional". busybox, dbus and systemd
+set PKG_PRIORITY="required".
+The effect of PKG_PRIORITY is unknown.
+
+Proposal:
+Deprecate the PKG_PRIORITY variable
+
+
+2 PKG_SITE, PKG_LONGDESC, PKG_SHORTDESC
+
+None of the current scripts use the PKG_SITE variable.
+create_addon is the only script that uses the PKG_LONGDESC and PKG_SHORTDESC
+variables.
+In other words, the PKG_LONGDESC and PKG_SHORTDESC are only used for
+addons. For other packages, the PKG_SITE, PKG_LONGDESC and PKG_SHORTDESC are
+merely informative and redundant.
+
+Proposal:
+Deprecate PKG_LONGDESC and/or PKG_SHORTDESC for normal packages
+
+
+3 PKG_AUTORECONF
+
+More than 330 packages set PKG_AUTORECONF="no", whereas about 230 packages set
+PKG_AUTORECONF="yes".
+build is the only script that uses the PKG_AUTORECONF variable.
+
+Proposal: 
+Modify build to set PKG_AUTORECONF="no" if PKG_AUTORECONF is not set in
+in package.mk
```

Comment:
You have a point.
For the sake of completeness, build merely checks if PKG_AUTORECONF is set to yes, ie no is already the default.